plugin-core/src/main/java/com/salesforce/bazel/eclipse/runtime/api/JavaCoreHelper.java

```diff
@@ -186,10 +186,9 @@ void setClasspathContainer(IPath containerPath, IJavaProject[] affectedProjects,
     IJavaProject getJavaProjectForProject(IProject project);
 
     /**
-     * Gets the list of all Java projects in the Workspace.
-     * @return
+     * Gets the list of all Bazel Java projects in the Workspace, <b>EXCLUDING</b> the Root Bazel WORKSPACE project. 
      */
-    IJavaProject[] getAllJavaProjects();
+    IJavaProject[] getAllBazelJavaProjects();
```

Comment:
In some places we were getting Java projects, in others we were explicitly checking for the Bazel nature.  In all cases, we don't actually need the root project.  So now we only call this method.